### DISCUSSION TITLE
How to delete pokemons from terminal

### DISCUSSION THREAD

**Discussion by archmaxxxing** [kRHYME7/Hyde-cli#91] [Q&A] [answered]
Hi, I'm complete noob in terms of configuration/customization and i don't have any idea how those pokemons are even done. I want to delete just them and keep everything else. Search discussions and didn't find anything.

**Comment by kRHYME7** [comment, marked as answer]
Remove the pokemon line of the ~/.zshrc file.

**Comment by erikldr** [comment]
in terminal: `sudo nano ~/.hyde.zshrc`
remove the line: `pokego --no-title -r 1,3,6`

**Reply by voidbournreloaded** [reply]
what happens when you input `sudo nano ~/.hyde.zhrc` and zero lines show up in micro/ nano?

**Reply by kRHYME7** [reply]
don't sudo it. the `root` user cannot touch `$USER` files.

**Reply by voidbournreloaded** [reply]
With or without sudo, no lines of code appear in the ui after typing in the specified command exactly

**Reply by kRHYME7** [reply]
`code  $ZDOTDIR/user.zsh`

that file is moved to `$ZDOTDIR/user.zsh`
